contact/contact.py

```diff
@@ -0,0 +1,48 @@
+"""
+ID: whuan2001
+LANG: PYTHON2
+TASK: contact
+"""
+def solution():
+    fin = open("contact.in", "r")
+    fout = open("contact.out", "w")
+    A, B, N = map(int, fin.readline().strip().split())
+    string = ""
+    for line in fin.readlines():
+        string += line.strip()
+    dic = {}
+    for lens in range(A, B + 1):
+        if len(string) - lens >= 0:
+            for i in range(len(string) - lens + 1):
+                if string[i : i + lens] not in dic:
+                    dic[string[i : i + lens]] = 0
+                dic[string[i : i + lens]] += 1
```

Comment:
You can use counter to replace dict

https://docs.python.org/2/library/collections.html#collections.Counter